### ISSUE TITLE
Screen reader is not announcing 'Name', 'Role' and 'State' information for the controls appearing in 'See more on Bing' pop up.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27866]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: [Telegram: Contact @SCTejTestbot](https://t.me/SCTejTestbot) on edge.
3. SCTestbot home screen will appear.
4. Activate on start button.
5. Navigate to edit filed enter 'accessibility' query and invoke it.
6. Navigate to See more on Bing control on response section.
7. Turn on Narrator.
8. Observe an issue that Screen reader is not announcing 'Name', 'Role' and 'State' information for the controls appearing in 'See more on Bing' pop up.

### Expected behaviour

 Screen reader should announce 'Name', 'Role' and 'State' information for the interactive controls.

### Actual behaviour

Screen reader is not announcing 'Name', 'Role' and 'State' information for the controls appearing in 'See more on Bing' pop up.

https://github.com/telegramdesktop/tdesktop/assets/93735775/ddb7eb39-d091-4187-a0fa-ef8aeec0d32e

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
